build.gradle

```diff
@@ -24,9 +24,12 @@ import java.util.regex.Pattern
 buildscript {
   repositories {
     gradlePluginPortal()
+    maven {
+      url = uri("https://plugins.gradle.org/m2/")
+    }
   }
   dependencies {
-    classpath 'com.github.johnrengelman:shadow:8.1.1'
+    classpath "io.github.goooler.shadow:shadow-gradle-plugin:8.1.7"
```

Comment:
This is an issue with the plugin: https://github.com/johnrengelman/shadow/issues/911